src/main/java/teammates/ui/template/FeedbackSessionsForm.java

```diff
@@ -2,19 +2,32 @@
 
 import java.util.List;
 
+import teammates.common.util.Url;
+
+
 /**
- * Data model for the form for creating a new feedback session.
+ * Data model for the form for creating/editing a feedback session.
  *
  */
 public class FeedbackSessionsForm {
 
+    private Url formSubmitAction;
+    private String submitButtonText;
+    
     // Default course id value
     private String courseIdForNewSession;
     
     private String feedbackSessionNameForSessionList;
     
+    private boolean isEditFsButtonsVisible;
+    private boolean isFeedbackSessionTypeEditable;
     // List of options for feedback session type
     private List<ElementTag> feedbackSessionTypeOptions;
+    
+    private Url fsDeleteLink;
+    private Url copyToLink;
```

Comment:
As far as I can tell, you're just using the `Url` constructor with its `toString()` method which do nothing more than treating it as a normal `String`.
```
public Url(String url) { this.urlString = url; }
public String toString() { return urlString; }
```